.github/workflows/pull-request-ci.yaml

```diff
@@ -171,15 +171,23 @@ jobs:
           echo "SSH_AGENT_PID=${SSH_AGENT_PID}" >> "${GITHUB_ENV}"
         timeout-minutes: 1
 
-      # set GKE secret. TBD: use vault instead
+      - name: Query vault for GKE secrets
+        id: vault
+        uses: hashicorp/vault-action@v2.0.1
+        with:
+          exportEnv: false
+          url: https://volt.cap.explore.suse.dev/
+          token: ${{ secrets.VAULT_TOKEN }}
```

Comment:
Huh, I thought `secrets.VAULT_TOKEN` didn't have the correct access, and we needed `secrets.VAULT_TOKEN_CLOUD_PROVIDERS` for that; see the scheduler workflow.

https://github.com/cloudfoundry-incubator/kubecf/blob/7762c65756fde37a126f1f5e701bb6056c93f55a/.github/workflows/pull-request-schedule.yaml#L40-L46

Reply:
Addressed with commit abcd4410dac0c7a0970ad5ea69f6b01bbf05a048